src/server/req_delete.c

```diff
@@ -1555,6 +1550,8 @@ post_delete_mom1(struct work_task *pwt)
 
 	preq_sig = pwt->wt_parm1;
 	rc = preq_sig->rq_reply.brp_code;
+	if (rc && pwt->wt_aux && (rc < PBSE_) && (pwt->wt_aux > PBSE_))
+		rc = pwt->wt_aux;
```

Comment:
what's this doing?

Reply:
It sets PBSE_ code if any other error is reported in the brp code. I'll add a comment.